### ISSUE TITLE
Discussion: build fatal error YogaKit.modulemap' not found on Apple M1 simulator

### ISSUE THREAD

**Issue by OuterCloud** [facebook/flipper#3950]
<img width="1400" alt="image" src="https://user-images.githubusercontent.com/16227179/181877531-6f1fa346-d738-4ed5-b0fc-42f330adf602.png">
Build failed after pod install
See the error below:
fatal error: module map file '/Users/tianyou.lan/Library/Developer/Xcode/DerivedData/ForFlipper-blfjjkxgwregxndruixckylstpuk/Build/Products/Debug-iphonesimulator/YogaKit/YogaKit.modulemap' not found

Apple M1
On simulator

**Comment by LukeDefeo** [comment]
Can you make sure you are opening the generated workspace and not the project file

**Comment by OuterCloud** [comment]
> Hi there, can you provide us with more information about what you are trying to achieve and what you have done to get to this state.

Hi, Luke,
Nice to meet you.
This is my app demo project which wants to integrate the flipper: https://github.com/OuterCloud/ForFlipper
Hope to get help from you.
Best regards.

**Comment by OuterCloud** [comment]
> Can you make sure you are opening the generated workspace and not the project file

Yes, I make a xcode project and open the project from the workspace file.

**Comment by OuterCloud** [comment]
@LukeDefeo I can build the Flipper/iOS/Sample successfully, but I can not build my project with flipper. The Podfiles are the same, what's the difference then?

**Comment by OuterCloud** [comment]
@LukeDefeo I really need your help, please help me. Thanks very much.

**Comment by tarkcelk** [comment]
did you solve it ?
